### ISSUE TITLE
repository is almost 2 months old. it's not super clear how to update it.

### ISSUE THREAD

**Issue by dysinger** [timbertson/opam2nix-packages#31]
anything after August isn't recognized by opam2nix-packages right now. I would like to be able to update my own fork at will.

**Comment by timbertson** [comment]
Yeah, I got a bit inundated. I'll try to get to this one soon, but I'm a
bit busy at the moment. Travis used to do it via cron, something must have
broken :/

**Comment by timbertson** [comment]
I tried bumping manually bit it looks like something's broken in the digest
mapping code, unfortunately. It'll take a bit of time to fix :/

**Comment by dysinger** [comment]
I tried my hand at fixing this yesterday. I was not successful immediately. Should we create an issue for it? I'm going to continue on it today.

**Comment by timbertson-zd** [comment]
I [fixed](https://github.com/timbertson/opam2nix/commit/19139d426a227a370806bed180c02b8cb329f136) the issue with digests yesterday (the opam2 repository format moved there URLs come from), so I _think_ the (temporary) `opam2-testing` branch should be up to date now:

https://github.com/timbertson/opam2nix-packages/tree/opam2-testing

I haven't had a chance to test it yet (thus I haven't pushed to the `opam2` branch), but please try it out and report back. Still need to fix travis before closing this issue, but hopefully it works for the moment.

**Comment by dysinger** [comment]
ok great. i'm working on it too today. I'll integrate.

**Comment by dysinger** [comment]
with my branches (P/R submitted on both opam2nix and opam2nix-packages) I can now build a complex bucklescript project which is a hybrid ocaml 4.02.3 + nodejs project. so I think this branch is on the right track. I can also _almost_ build a complex 4.06.1 reasonml web server
